addons/udes_stock/models/stock_location.py

```diff
@@ -134,6 +134,41 @@ def _domain_height_category(self):
         see u_countable field.""",
     )
 
+    u_heatmap_data_updated = fields.Datetime(
+        string="Heatmap data updated",
+        help="""
+            The date when the location name, co-ordinates or size was last changed. 
+        """,
+        default=fields.Datetime.now,
+    )
+
+    def set_u_heatmap_data_updated(self, vals):
```

Comment:
not 100% on the name, initially had `update_u_heatmap_data_updated()` but that also felt a bit weird

Reply:
sorry, do you mean good as is, or you'd like it changed to `update_u_heatmap_data_updated()`?